### PR TITLE
Cleaned the code

### PR DESCRIPTION
**Pull request by CleanMachine1**
This is untested however, as I was just browsing Github I came across this and saw an easy fix.

Please do test before merging.

### CODE REVIEW

File: musicplayer.py [hunk 1]
```diff
@@ -57,77 +57,12 @@ def stop(self):
...
-                mixer.init()
-                mixer.music.set_volume(0.8)
-            elif self.bttn_clicks1==9:
-                mixer.init()
-                mixer.music.set_volume(0.9)
-            elif self.bttn_clicks1==10:
-                mixer.init()
-                mixer.music.set_volume(1.0)
-                
+            float(button_clicks) == self.bttn_clicks
```
Comment (CleanMachine1) [inline musicplayer.py]:
```suggestion
            float(button_clicks) = self.bttn_clicks
```

**Comment by CleanMachine1** [comment]
Theres an error, however its too late my time to fix it.

I'll fix it when I am next available.

**Comment by Raspberry-Pi-Reviews** [comment]
shoot sorry for not responding I havent been on my pc in a while and thanks for cleaning up the code if you did.

**-RaspberryPIReviews**

**Comment by CleanMachine1** [comment]
I am going to close since I am probably not going to do it, I'll be honest, however as you can clearly see the excess amount of code isn't efficient.

Having code for every single possible outcome is not a good idea since it makes code unreadable

See the following blocks of code:

bad code:
```python
var1 = input("Enter a letter")
if var1 == "a":
  print("a")
elif var1 == "b":
 print("b")
....
```

better code:
```python 
var1 = input("Enter a letter")
print(var1)
```

**Pull request closed without merging**